rules/k8s_audit_rules.yaml

```diff
@@ -102,6 +102,12 @@
 - macro: role
   condition: ka.target.resource=roles
 
+- macro: secret
+  condition: ka.target.resource=secrets
+
+- macro: req_service_account_token
```

Comment:
Mark, we cannot set the audit level to `Request` or `RequestResponse` or the secret data will be recorded in the audit log.

Reply:
This isn't controlling the audit level though. This macro just looks for the type of the object for the request being a service account token.

You are right, though, that the recommended audit policy logs secrets at the RequestResponse level: https://github.com/falcosecurity/falco/blob/master/examples/k8s_audit_config/audit-policy.yaml#L59. How about we also change the recommended policy to log secrets at Metadata level and then change these rules to exclude requests starting with "system:" ?

Reply:
The macro reminds me that it is must from audit level `Request` or `RequestResponse`. How about having a `trusted_users` list?

Reply:
Yeah I'll change it to a users list.